paket.dependencies

```diff
@@ -1,16 +1,29 @@
-source https://www.nuget.org/api/v2
-framework: net47
-storage: none
-nuget FSharp.Core redirects:force
+framework: auto-detect
+
+source https://api.nuget.org/v3/index.json
+
 nuget Elmish prerelease
+nuget FSharp.Core >= 4.1.17 redirects: force
+
+
+group Build
+
+  source https://api.nuget.org/v3/index.json
+
+  nuget Fake.Core.Target
+  nuget Fake.Core.ReleaseNotes
+  nuget Fake.DotNet.Cli
+  nuget Fake.DotNet.Paket
+  nuget Fake.IO.FileSystem
+
 
-group build
-framework: >= net45
-source https://www.nuget.org/api/v2
-nuget FAKE
-nuget FSharp.Core redirects:force
+group Test
 
-group sample
-source https://www.nuget.org/api/v2
+  source https://api.nuget.org/v3/index.json
 
-nuget FsXaml.Wpf
\ No newline at end of file
+  nuget Hedgehog
+  nuget Hedgehog.Experimental
+  nuget Microsoft.NET.Test.Sdk
+  nuget Unquote
+  nuget xunit.core
+  nuget xunit.runner.visualstudio version_in_path: true
```

Comment:
Does this work? At least in VS 2013 test adapters had to be located in `packages/` directly.

Reply:
Works great in VS2017.